antora.yml

```diff
@@ -14,104 +14,189 @@ asciidoc:
     serverlessoperatorname: OpenShift Serverless Operator
     certmanageroperatorname: OpenShift Cert-Manager Operator
     serverlessproductname: OpenShift Serverless
+
+    # 
+    # Serverless Logic - Names, labels and similar
+    #
     product_name: OpenShift Serverless Logic
-    kogito_version_redhat: 9.99.0.redhat-00007
+    kogito_version_redhat: 9.100.0.redhat-00004
+    kogito_branch: 9.100.x-prod
     operator_name: Serverless Logic Operator
+    operator_installation_namespace: sonataflow-operator-system
+    operator_controller_config: sonataflow-operator-controllers-config
     quarkus_platform: com.redhat.quarkus.platform
-    kogito_sw_ga: >-
-      org.kie.kogito:kogito-quarkus-serverless-workflow
-    quarkus_version: 3.2.9.Final-redhat-00003
-    quarkus_platform_version: 3.2.9.Final-redhat-00004
+    kogito_sw_ga: kogito-quarkus-serverless-workflow
+    data_index_ref: Data Index
+    workflow_instance: workflow instance
+    workflow_instances: workflow instances
+    operator_openshift_keyword: 'Serverless Logic'
+    operator_openshift_catalog: logic-rhel8-operator
+    operator_k8s_keyword: sonataflow
+    operator_k8s_subscription: my-sonataflow-operator
+    osl_kn_cli_imagename: registry.redhat.io/openshift-serverless-1/kn-workflow-cli-artifacts-rhel8
+    kogito_devservices_imagename: registry.redhat.io/openshift-serverless-1/logic-data-index-ephemeral-rhel8
+    sonataflow_devmode_imagename: registry.redhat.io/openshift-serverless-1/logic-swf-devmode-rhel8
+    sonataflow_builder_imagename: registry.redhat.io/openshift-serverless-1/logic-swf-builder-rhel8
+    sonataflow_dataindex_ephemeral_imagename: registry.redhat.io/openshift-serverless-1/logic-data-index-ephemeral-rhel8
+    sonataflow_dataindex_postgresql_imagename: registry.redhat.io/openshift-serverless-1/logic-data-index-postgresql-rhel8
+    sonataflow_devmode_devui_url: /q/dev-ui/org.apache.kie.sonataflow.sonataflow-quarkus-devui/
+    serverless_logic_web_tools_name: Serverless Logic Web Tools
+    serverless_workflow_vscode_extension_name: Openshift Serverless Logic Workflow Editor
+    kie_kogito_examples_repo_name: kogito-examples
+
+    # Jobs service image and links
+    jobs_service_image_ephemeral: registry.redhat.io/openshift-serverless-1/logic-jobs-service-ephemeral-rhel8
+    jobs_service_image_ephemeral_name: logic-jobs-service-ephemeral-rhel8
+    jobs_service_image_ephemeral_url: https:registry.redhat.io/openshift-serverless-1/logic-jobs-service-ephemeral-rhel8
+    jobs_service_image_postgresql_name: logic-jobs-service-postgresql-rhel8
+    jobs_service_image_postgresql: registry.redhat.io/openshift-serverless-1/logic-jobs-service-postgresql-rhel8
+    jobs_service_image_postgresql_url: https://registry.redhat.io/openshift-serverless-1/logic-jobs-service-ephemeral-rhel8
+    jobs_service_image_usage_url: https://github.com/kiegroup/kogito-images/tree/9.100.x-prod#jobs-services-all-in-one
```

Comment:
Probably it's better to unify URL formats (using in all of them the same protocol or none).
Also take into account that now is `catalog.redhat.com`

Reply:
Yea should be catalog URL